src/Testing/TestCase.php

```diff
@@ -339,6 +339,21 @@ public function getFileHelper(): FileHelper
 		return self::getContainer()->getByType(FileHelper::class);
 	}
 
+	/**
+	 * Provides a DIRECTORY_SEPARATOR agnostic assertion helper, to compare file paths.
+	 *
+	 * @param string $expected
+	 * @param string $actual
+	 * @param string $message
+	 */
+	protected function assertSamePaths(string $expected, string $actual, string $message = ''): void
```

Comment:
I put his helper in the base-class because I guess it can be re-used to fix a few of the remaining 6 windows-only testsuite errors

Reply:
I agree with that, but try using FileHelper instead of str_replace.

Reply:
If it doesn't succeed, I'm gonna merge this as it is.

Reply:
using normalizePath will make the test error on windows travis CI

Reply:
Looks like it passes.